app/assets/stylesheets/custom.scss

```diff
@@ -46,8 +46,9 @@ input[type=checkbox] {
 }
 
 #flash-success {
-  background-color: rgba(0, 188, 2, 0.08);
-  border: 2px solid rgba(0, 188, 2, 0.4);
+  background-color: rgba(70, 79, 113, 0.4);
+  border: 2px solid $grey;
+  padding-left: 5px;
```

Comment:
Properties should be ordered background-color, border, border-radius, padding-left